server.js

```diff
@@ -24,8 +26,26 @@ app.get('/test', (request, response) => {
 
 })
 
-app.get('/book', (request, response) => {
-  
-})
+app.get('/book', handleBooks);
+
+
+async function handleBooks(request, response) {
+  let searchQuery = {};
+  if (request.query.location) {
+    searchQuery = {
+      location: request.query.location
+    }
+  }
+  try {
+    let booksDB = await Book.find(searchQuery);
```

Comment:
Good job on this!

In class Ryan demoed something that looked this for Cats. In this case though, we need the user email. 
Grab a TA or Ryan and see what we can do for lines 34- 40. 

- Task card 1 (last bullet in the list) asks us to:  Retrieve all of the books belonging to requesting user. To do this, we need to use the user email on the req object.👍